tests/EndToEnd/ReadmeExamples.php

```diff
@@ -74,6 +74,26 @@ public function testTemporaryFilesAreDeleted()
 		$this->assertEquals(false, file_exists($ocr->command->getOutputFile(false)));
 		$this->assertEquals(false, file_exists($ocr->command->getOutputFile(true)));
 	}
+	
+	public function testOutputFileIsCreated()
+	{
+		// https://github.com/thiagoalessio/tesseract-ocr-for-php/issues/174
+		// Tesseract can not extract Files in version 3.02
+		if ($this->isVersion302()) $this->skip();
+
+		$outputFileName = "/output.hocr";
+		$outputFolderName = sys_get_temp_dir() . "/output";
+		$ocr = new TesseractOCR("{$this->imagesDir}/text.png");
+		$ocr->setOutputFile( $outputFolderName . $outputFileName);
+		$ocr->configFile('hocr');
+		$ocr->run();
+
+		$this->assertEquals(true, file_exists( $outputFolderName . $outputFileName));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Space after opening parenthesis of function call prohibited](https://app.codacy.com/app/thiagoalessio/tesseract-ocr-for-php/pullRequest?prid=4607615)